README.md

```diff
@@ -275,6 +277,39 @@ var swiper = React.createClass({
 
 > More ScrollResponder info, see: https://github.com/facebook/react-native/blob/master/Libraries/Components/ScrollResponder.js
 
+#### Flex sizing
+
+To create a flexibly sized swiper, use a wrapper with the `onLayout` prop, as below:
+```jsx
+// creating our component
+class Layout extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      height: -1,
+      width: -1
+    }
+  }
+
+  measureLayout({nativeEvent}) {
+    const height = nativeEvent.layout.height;
+    const width = nativeEvent.layout.width;
+    this.setState({height, width});
```

Comment:
`setState` fails when `measureLayout` is called from `onLayout` because the context is not the component.

`measureLayout` has to be bound in the constructor, or called via an arrow function for this example to work.